### ISSUE TITLE
Revise Vision Statement

### ISSUE THREAD

**Issue by benhylau** [tomeshnet/documents#92]
As part of website revamp discussion, we decided to revisit what is on our [About page](https://tomesh.net/about/). We should see if the Vision still reflects our current focus. Some of the points raised include:

* We are doing a lot of work outside of what is traditionally considered a mesh network (e.g. peer-to-peer applications)
* Our focus is currently quite different from NYC Mesh, referencing their definition is a little misleading for visitors

Given this is a change in one of the most significant outward facing messaging, we should decide on  the copy collaboratively. This ticket is to facilitate that consensus process. The output should include:

- [ ] Copy to replace our current About page
- [ ] Short one or two line vision to replace the current `We are building a mesh network in Toronto` on our landing page

**Comment by benhylau** [comment]
Suggesting to use https://pol.is

**Comment by makew0rld** [comment]
I believe cryptpad has polls as well, whatever's easier.

**Comment by benhylau** [comment]
Pol.is is not just a poll. It is a very elaborate tool that allows participants to keep evolving the conversation and establish common grounds, that may end up being very different from the original seed questions in the poll.

I am collecting some possible poll questions that came out of chats:
https://cryptpad.fr/code/#/2/code/edit/PHtY2VfOeF7txAmnzs7YyeK8/

**Comment by dasanchez** [comment]
I had a good talk with ryan_f and structur in our last meetup about how to make this an open and inclusive activity. 

First off, to re-state the scope:
- Produce a mission statement for Toronto Mesh. This will replace the `we are building a mesh network in Toronto` statement currently shown in the landing page.
- Update the About Us page.

I have the following plan in mind:

- [X] Request comments from everyone looking to contribute to the new mission statement. I have started a cryptpad for this purpose here:
https://cryptpad.fr/pad/#/2/pad/edit/CIx-f9-0GsSWlBa4CK6dNm6c/
And give it two weeks for everyone to share their thoughts.

- [X] Identify the main themes and common goals established by the comments and produce a set of questions we can use to guide a conversation. This content would also be used later on to update the About Us page.

- [x] Poll people online.
@benhylau, are you a pol.is user? I saw the "Request a Demo" button and just groaned. And it's either that or sign in with facebook 😞 
Somebody mentioned loomio in scuttlebutt...I liked it, but the free plan only allows 10 people a group.

- [x] Meet face-to-face to produce several options for the mission statement.
I see this as a 2-3 hour activity (probably longer). We would release all our notes as meeting minutes in the `documents` repo. **Edit**: If possible, we can do this during the planning session.

- [x] Publish mission statement and submit PR for About Us page.

What do you think?

**Comment by benhylau** [comment]
I think this is great. My requests:

* Once this process is complete, the consultation cryptpad gets moved to a git repo, making it a .md cryptpad now may save us future troubles :) so we can reference how we arrived at the new statement

* >@benhylau, are you a pol.is user? I saw the "Request a Demo" button and just groaned. And it's either that or sign in with facebook 😞

    I have used it before as a user, but don't have an account. Never facilitated a process, but am somewhat familiar with how it works from looking at this case study https://blog.pol.is/pol-is-in-taiwan-da7570d372b5

    It's unfortunate their on-board experience is quite restrictive based on the options you described. I think it's a tool that allows us to do what you proposed in an automated way, but given small size of our community I think it's okay to do it manually, and your more manual process also allow for better paper record.

If you do choose to try pol.is the best way is probably to request a demo, but I feel good about your plan, with or without pol.is.

**Comment by dasanchez** [comment]
> Once this process is complete, the consultation cryptpad gets moved to a git repo, making it a .md cryptpad now may save us future troubles :) so we can reference how we arrived at the new statement

Done! 👍

**Comment by benhylau** [comment]
Also we can basically do a pol.is-like process, bootstrapping off of GitHub identities. Just open a new issue, and any of us can seed a new question as a comment, e.g.:

>Could I think of the idea of Toronto Mesh as an idea incubator and shared digital workspace with groups working together and independently? With a share commons of goals/vision?

Then we just do GitHub's 👍 👎 on each issue.

**Comment by dasanchez** [comment]
Well it has taken me a while, but I think I have something we can use to move this discussion forward. Apologies for the wait 😅 

I read, summarized, and categorized the comments submitted through [CryptPad](https://cryptpad.fr/pad/#/2/pad/edit/CIx-f9-0GsSWlBa4CK6dNm6c/) and [GitHub](https://github.com/tomeshnet/documents/pull/99/files), and prepared a draft for the updated Vision section of the About page. This draft includes five options for the mission statement, which would be worded differently on the home page. For instance, the following would be used in the About page:

_This is our mission: **to prototype the next generation of the internet.**_

But the home page would read:

_We are prototyping the next generation of the internet_

Please [read the draft here](https://cryptpad.fr/code/#/2/code/edit/xbG3NnptacNwrMpkWxuuqEEd/) and post your comments here.

I am also posting the mission statement options below- vote for the one you like the most and leave some feedback! Like @benhylau said, let's try doing 👍 and 👎 

My hope is that we can trim our options to three or less statements we can vote on during the planning session. If somehow are all happy with one by then, we can use the planning session to announce it!

**Comment by dasanchez** [comment]
### Mission statement # 1:

Prototype the next generation of the internet

**Comment by dasanchez** [comment]
### Mission statement # 2:

Empower communities to create better networks that rely on open source hardware, peer-to-peer technologies, and digital literacy.

**Comment by dasanchez** [comment]
### Mission statement # 3:

Build alternatives to the centralized web.

**Comment by dasanchez** [comment]
### Mission statement # 4:

Empower communities through decentralization.

**Comment by dasanchez** [comment]
### Mission statement # 5:

Empower those dedicated to building community networks.

**Comment by benhylau** [comment]
Can we vote more than one?

**Comment by dasanchez** [comment]
@benhylau Yes, we can vote for more than one!

**Comment by Shrinks99** [comment]
I think #2 is very accurate although a little long.  Maybe we can discuss shortening it if it gets picked?  Statement #3 also makes sense to me.

**Comment by dasanchez** [comment]
\# 2 is verbose indeed! I'd say we're mostly voting for the message, so we can adjust the wording- any suggestions? Post an alternative and I may switch my vote :wink:
I'll go first!

**Comment by dasanchez** [comment]
## Mission statement # 6:

Empower communities to create networks that rely on open source hardware, peer-to-peer technologies, and digital literacy.

**Comment by dcwalk** [comment]
Thanks for all this work @dasanchez ! 

I think there is something going on with the parallel structure in *Mission #2 or #6* around digital literacy. Is the point that the networks rely on digital literacy (I think almost everything digital does?) or that these networks better promote it?

I'm also not sure about the "empower communities" because that seems to imply a sort of distance (we are doing this for other groups not ourselves?) that I'm not sure Id want to imply. That said. I think #2 and #6 are close.

Also, I feel like I have to say: which communities?

# Mission statement #7:

To explore building networks using peer-to-peer technologies and open source hardware that empower communities and increase digital literacy.
<which communities?>

**Comment by dcwalk** [comment]
# Mission statement #8:
Empowering communities <which>? and increasing digital literacy through networks that rely on peer-to-peer technologies and open source hardware.

**Comment by dasanchez** [comment]
_I'm also not sure about the "empower communities" because that seems to imply a sort of distance (we are doing this for other groups not ourselves?) that I'm not sure Id want to imply_
I would interpret it as we are doing this for ourselves, but we welcome any other group to use our resources and ask for assistance.

_which communities?_
I'd say any group of people looking for ways to get connected. Maybe I'm looking for advice on how to build a local communication system with my neighbours, or maybe I'm just looking into starting an online discussion without relying on Reddit/Twitter/Facebook. Thanks to Toronto Mesh I can get guidance on both deploying a mesh network and joining SSB/Aether.

**Comment by dasanchez** [comment]
_Is the point that the networks rely on digital literacy (I think almost everything digital does?) or that these networks better promote it?_
I think it's more of the former- digital literacy is a requirement for building a network. Taking it a step further, I would say the "empowering" implies providing digital literacy.

**Comment by dasanchez** [comment]
## Mission statement # 9:

Empower communities to create networks that rely on open source and peer-to-peer technologies.

**Comment by benhylau** [comment]
How do we feel about the word _self-sovereign_ in the infrastructural and data context?

It embeds a lot of our values like agency, privacy, community empowerment... but could perhaps be too restrictive in a mission statement?

**Comment by makew0rld** [comment]
I think that's a really good way to describe what we do, creating software and networks that effectively mean self-sovereignty: it's no longer corporations or the government that controls your traffic, it's *you*. (Or your community 😉) 

I don't think it's restrictive at all for a mission statement, it gives us a lot of freedom.

**Comment by dasanchez** [comment]
@benhylau it's the first time I hear about self-sovereign, but an online search mostly returns "self-sovereign _identity_" definitions from blockchain-related articles. Is that what you're talking about? If that's the case, I would be OK with adding it to the Vision text, and expanding on the term with something like what you and @makeworld-the-better-one just mentioned.

I would like to make the mission statement as accessible to non-techies as possible, and part of me feels like "peer-to-peer" is already an esoteric term (though easily explained).

**Comment by makew0rld** [comment]
@dasanchez I see how it could be a little confusing, maybe a better word is independent, or self-governing. Maybe one of those words could precede the work network in Mission Statement 6

**Comment by benhylau** [comment]
@dasanchez yes that is also my concern, that the term especially used in this context is not well established. It also can be misinterpreted as something we do not intend. And yes, _self-sovereign identity_ is part of it, as in cjdns, SSB, IPNS (how we generate key pairs "permissionlessly") and that is a large part of what we do, but @makeworld-the-better-one this is also why I mentioned it may be a restrictive in a mission statement, since most people may not care about that type of design.

@makeworld-the-better-one I get your point about self-sovereign as applied to a community to their technical infrastructure, that is how I want to use the term, but also understand that is not an accessible definition at all as @dasanchez pointed.

**Comment by dasanchez** [comment]
I updated the Vision text with the following:

A community-owned network asks individuals to understand the technologies they use and to make decisions based on their community needs, rather than those of corporations. _In return, users gain self-sovereignty from services they do not own nor control._

**Comment by makew0rld** [comment]
Where did you add it? Also, I find that wording confusing, I would maybe define community networks differently. I would change the last sentence as well, maybe something about how they gain self-sovereignty instead of using services they do not control, etc

**Comment by dasanchez** [comment]
@makeworld-the-better-one you're right. I moved the bit about self-sovereignty to the section on applications instead ([see the draft here](https://cryptpad.fr/code/#/2/code/edit/xbG3NnptacNwrMpkWxuuqEEd/)):

We are adopting applications built on open, distributed systems that rely on peer-to-peer communication rather than centralized systems. _These applications give us an opportunity to develop our self-sovereignty by giving us better control over our personal data, our communications, and the content we create and share._

Am I getting warmer now? @benhylau?

**Comment by benhylau** [comment]
>These applications give us an opportunity to develop our self-sovereignty by giving us better control over our personal data, our communications, and the content we create and share.

Suggest:

_These applications give us, as participants of a network, an opportunity to develop self-sovereignty over our personal data, our communications, and the content we create and share._

To better define _us_ and emphasize the scope of the _self-sovereignty_.

**Comment by dcwalk** [comment]
Hi all! 

Just chiming in a bit late, it might work well to have some of this conversation in person at the planning meeting on Thursday?

I have some thoughts (or just general pause) about including language of "self-sovereignty" given its lineage from blockchain/identity management space. And also the wider political currents a term like "sovereignty" comes from (I think [Hobbes' Leviathan](https://en.wikipedia.org/wiki/Leviathan_(Hobbes_book)) and [other thinkers](https://plato.stanford.edu/entries/sovereignty/)) stir up many connections to associated concepts that I'm not sure I want to claim.

Specifically in the context of community ownership of infra, there are many established related terms/phrasings that are quite close: autonomy, "tech sovereignty," self-reliance...etc... so I'm curious what you all see "self-sovereignty" adding?

A bit of a ramble led me through a lot of [identity](http://www.lifewithalacrity.com/2016/04/the-path-to-self-soverereign-identity.html) and [blockchain](https://blog.selfkey.org/the-decentralized-identity-economy-f3dbfc9a3c3c), as well as some pretty [intense](http://selfsip.org/dialogues/ghs/SSIPcritique_3.html) projects and [few individualist-oriented anarchists](https://books.google.ca/books?id=bk-aaMVGKO0C&pg=PA101&lpg=PA101&dq=%22each+person+is+the+owner+of+herself,+or+is+self-sovereign,%22&source=bl&ots=EMvJ50o9MX&sig=X07j3rvMcr65HZzGuajXMCHg_go&hl=en&sa=X&ved=2ahUKEwiwxdzopeHeAhULjoMKHWEIDZ8Q6AEwAHoECAMQAQ#v=onepage&q=%22each%20person%20is%20the%20owner%20of%20herself%2C%20or%20is%20self-sovereign%2C%22&f=false) ([also](https://www.jstor.org/stable/j.ctt14bs03s)).

---
In direct response to:
> "develop self-sovereignty over our personal data"

I think "assert sovereignty over our personal data" is much cleaner, or even more plain language options:
- "take control over our personal data" (and then reworking the structure of the serial list)
- "These applications empower us to have better control over our personal data, our communications, and the content we create and share"
- "These applications allow us to create autonomous networks where we have better control over our personal data, our communications, and the content we create and share"

etc...

---

Final stray thought--
We as Toronto Mesh haven't used the language of data too much, and that feels like a bit of a departure from current projects (which don't focus on data holding it seems so much as different forms of data transit), just something to mull on.

**Comment by benhylau** [comment]
I think our current projects already departed from transit, as we integrate and form conversations around ipfs / ssb / soon aether on our node software. I feel it's important to emphasize that component, and that data focus actually highlights one of our major differences with traditional mesh networks.

I favour this over alternatives:

>assert sovereignty over our personal data

Even though "self-sovereignty" better describes the permissionless nature of some software we employ today, I think as a group we care about the broader mission. I also don't want to use "better control over" because that is too ambiguous. For example, turning on more privacy settings on Facebook can be "better control over", but we are not advocating that. We are quite specifically trying to address this through data and infrastructural ownership, which I think is better described in the sovereignty framing.

**Comment by dcwalk** [comment]
data and infrastructural ownership != sovereignty... which typically references a form of power/authority witin a set space (and different theorizing/approaches to sovereignty imply answers about like how far that power extends, how much power there is, and who has the power).

The concepts are obviously close and overlapping, but--in ways I was trying to gesture at above--also distinct.

**Comment by benhylau** [comment]
@dcwalk but does `ownership + agency => sovereignty`?

What: "our personal data, our communications, and the content we create and share"
Who: "participants of a network"
How far: Depends? This one is not homogeneous.

**Comment by dcwalk** [comment]
hah, sure yeah @benhylau, fair point!

I guess what I've been waving at (confusingly) above is like maybe I see it as:
`ownership + agency + x + y => sovereignty`
but also
`ownership + agency + x + y => a`

and just wondering if `x`, `y` align with the most of us, and whether there is an `a` that is a better fit. But I've taken up too much space on this so I'll shut up!

**Comment by dasanchez** [comment]
Those links @dcwalk posted make a great point. I really don't want to ask people to go and read several manifestos and papers in order to understand two words in our About Us page. I liked this one the best:
```These applications empower us to have better control over our personal data, our communications, and the content we create and share.```
